### PR TITLE
fix README for mpas-bundle

### PR DESCRIPTION
**Pull request by snyderdad**
TYPE: bug fix

KEYWORDS: mpas-bundle, README

Jamie noticed that the README instructions are out of date, asking users to  "cd mpasjedi" before running ctest.  This replaces "mpasjedi" with "mpas-jedi" in two places.

LIST OF MODIFIED FILES: README

### CODE REVIEW

**Comment by snyderdad** [comment]
Presumably, these changes should also go back into JCSDA-internal/develop.  If someone can advise me of the best practice for accomplishing that, I'd appreciate it.  (Simply merge JCSDA/master back to JCSDA-internal/develop?)

**Review by liujake — APPROVED** [review]

**Comment by byoung-joo** [comment]
> Presumably, these changes should also go back into JCSDA-internal/develop. If someone can advise me of the best practice for accomplishing that, I'd appreciate it. (Simply merge JCSDA/master back to JCSDA-internal/develop?)


Based on @mmiesch 's comment https://github.com/JCSDA-internal/mpas-bundle/issues/117#issuecomment-928272244 , 
we might need a separate PR for  `JCSDA-internal/develop`.  Is this correct, @mmiesch ?

**Comment by byoung-joo** [comment]
Should `JCSDA-internal` at 9th line be replaced with `JCSDA`, too ?

**Review by byoung-joo — APPROVED** [review]

**Comment by snyderdad** [comment]
@byoung-joo, I made the correction you pointed to.

Now that I've actually stopped to think, it seems to me that we may not want to publish these instructions publicly at all.  The first tutorial covers similar material that's more relevant to public users (i.e. build in the container).  @byoung-joo, @liujake do you agree?

fv3-bundle has a simplified [README.md ](https://github.com/JCSDA/fv3-bundle/blob/master/README.md) on JCSDA, while on JCSDA-internal the [README](https://github.com/JCSDA-internal/fv3-bundle/blob/develop/README.md) has instructions for HPC builds (like here).

**Comment by byoung-joo** [comment]
Oh, that makes sense. In the public repository, it would be enough to refer our first tutorial: Build and Test JEDI-MPAS.

**Comment by snyderdad** [comment]
Closing and will open another PR with content like that of JCSDA/fv3-bundle.

**Comment by mmiesch** [comment]
Yes, we should decide on what we want to do here before we merge because after we merge, we may want to define a new tag of 1.0.1. 

I see another problem - the default build type is not release; it is RelWithDebInfo.

As we see here, the disadvantage of putting build instructions in a README is that one has to remember to keep them up to date.   I would recommend linking to the [generic build instructions for a bundle](https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/using/building_and_running/building_jedi.html) and the [build tutorial](https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/learning/tutorials/level2/dev-container_mpas_jedi.html) for general build instructions.  But, it still may be good to mention mpas-specific things here, like the use of the `env-setup` scripts.

**Comment by snyderdad** [comment]
@mmiesch, I didn't see your comment before closing this PR and opening another with a README.md like that for fv3-bundle.

In that PR, I originally had a link to the JEDI-MPAS build tutorial, as you suggest.  But I had a confidence crisis about how to link to specific pages in the documentation.  Is it sufficient to just navigate to the desired page, then cut and paste the address from the browser bar?  (Like: https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/learning/tutorials/level2/dev-container.html)

**Comment by byoung-joo** [comment]
@mmiesch, I have several questions.

> I see another problem - the default build type is not release; it is RelWithDebInfo.

In the `CMakeLists.txt` file of master branch, the default build type is set as `Release`. https://github.com/JCSDA/mpas-bundle/blob/7aa2bd3c19e677131189858c5cf78450216848a9/CMakeLists.txt#L16

Do you mean that this should be `RelWithDebInfo` ?

Also, in the [JEDI-MPAS build tutorial](https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/learning/tutorials/level2/dev-container_mpas_jedi.html#step-2-build-mpas-bundle), the command line instruction uses `ecbuild --build=Release ../mpas-bundle`.


If we want to define a new tag, I'd like to remove following two lines (rttov-related) because they are not available to public users.
https://github.com/JCSDA/mpas-bundle/blob/7aa2bd3c19e677131189858c5cf78450216848a9/CMakeLists.txt#L35-L36

**Comment by mmiesch** [comment]
@byoung-joo - you are correct - I did not look closely.  The default build type for ecbuild is RelWithDebInfo but I see this is overridden by that line in mpas-bundle.  So, disregard my comment.  The default build type is release and that is fine.

I'm ok with removing rttov.  I think we kept it in because external users might want to use it but even if they have the code they don't have the JCSDA repo and there are no instructions on how to implement it.  So, I think it's best to leave it out.  For advanced users who want to use it, we can guide them on the forum.

**Comment by mmiesch** [comment]
> @mmiesch, I didn't see your comment before closing this PR and opening another with a README.md like that for fv3-bundle.
> 
> In that PR, I originally had a link to the JEDI-MPAS build tutorial, as you suggest. But I had a confidence crisis about how to link to specific pages in the documentation. Is it sufficient to just navigate to the desired page, then cut and paste the address from the browser bar? (Like: https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/learning/tutorials/level2/dev-container.html)

Either way - a minimal readme like for fv3-bundle works too.  And yes, you can link to specific pages as you say.   For the master branch it might be better to replace "latest" with the "1.2.0" because the former is always changing, eg:  https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/1.2.0/learning/tutorials/level2/dev-container.html.   And, as I have said before, I think it generally better to link to the [top-level tutorials page](https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/1.2.0/learning/tutorials/index.html) than the specific tutorial.  But, in some contexts, like here where you're talking specifically about how to build mpas-bundle, then it might be appropriate to link the specific bundle.  

All of this is a judgement call and we can accommodate whatever you want to do - as long at it is accurate!

**Pull request closed without merging**
